package.json

```diff
@@ -47,7 +47,9 @@
     "prettier-plugin-solidity-refactor": "^1.0.0-alpha.12",
     "solc": "0.5.8",
     "truffle": "^5.0.2",
-    "truffle-contract": "^4.0.17",
-    "truffle-hdwallet-provider": "^1.0.0-web3one.1"
+    "@truffle/contract": "^4.1.2"
+  },
+  "dependencies": {
+    "@truffle/hdwallet-provider": "^1.0.27"
```

Comment:
You are also moving it from dev dependencies to dependencies, is this change intended?

Reply:
yes, see description:
But it should work out of the box for most projects, as I included the new npm package @truffle/hdwallet as a dependency, and not as a dev-dependency.